network/change_ssid.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#!/bin/bash
+
+set -e
+
+FILECONDITION="/etc/sysconfig/connman"
+SSID=$1
+PASSWORD=$2
+
+mkdir -p $(dirname $FILECONDITION)
```

Comment:
This is to tell the firstboot-connman if it should configure the network, which happens during first boot. I don't think you need to do anything here.  You are creating the dir, but you should also call a `touch $FILECONDITION` in the end.